src/main/java/com/comcast/zucchini/ZucchiniShutdownHook.java

```diff
@@ -117,10 +120,14 @@ public void run() {
                     rptName = System.getenv(NAME_ENV_VAR);
 
 
-                ReportBuilder reportBuilder = new ReportBuilder(pathList, html, "", rptName + version, "Zucchini" + version, true, true, true, false, false, "", false);
-                reportBuilder.generateReports();
+                Configuration reportConfig = new Configuration(html, "Zucchini" + version);
+                reportConfig.setBuildNumber(rptName + version);
+                ReportBuilder reportBuilder = new ReportBuilder(pathList, reportConfig);
+                Reportable reportable = reportBuilder.generateReports();
+                FileUtils.copyDirectory(new File(html, ReportBuilder.BASE_DIRECTORY), html); // move the cucumber base directory into zucchini
```

Comment:
Looks like the new report forces all files in a hard-coded base directory, so here I copy all the files to our existing zucchini-reports directory.

Reply:
Why not use `FileUtils.moveDirectory`?

Reply:
The structure is like:
`target/zucchini-reports/cucumber-html-reports/*`

So we want to move everything in `target/zucchini-reports/cucumber-html-reports/*`
to the base folder of `target/zucchini-reports`

I tried move, but since zucchini-reports already exists it fails. A copy does a merge of the files, and then I do a delete.